src/components/NotificationsDrawer/DrawerPanel.tsx

```diff
@@ -0,0 +1,515 @@
+              title: bundle.displayName,
+              value: bundle.name,
+            }))
+          );
+        }
+      } catch (error) {
+        console.error('Failed to fetch filter configuration:', error);
+      }
+    };
+    getNotifications();
+    fetchPermissions();
+    fetchFilterConfig();
+    return () => {
+      mounted = false;
+      unregister();
+    };
+  }, []);
+
+  const filteredNotifications = useMemo(
+    () =>
+      (activeFilters || []).reduce(
+        (acc: NotificationData[], chosenFilter: string) => [
+          ...acc,
+          ...notifications.filter(({ bundle }) => bundle === chosenFilter),
+        ],
+        []
+      ),
+    [activeFilters]
+  );
+
+  const onNotificationsDrawerClose = () => {
+    setActiveFilters([]);
+    toggleDrawer();
+  };
+
+  const onUpdateSelectedStatus = (read: boolean) => {
+    axios
+      .put('/api/notifications/v1/notifications/drawer/read', {
+        notification_ids: selectedNotifications.map(
+          (notification) => notification.id
+        ),
+        read_status: read,
+      })
+      .then(() => {
+        selectedNotifications.forEach((notification) =>
+          updateNotificationRead(notification.id, read)
+        );
+        setIsDropdownOpen(false);
+        updateAllNotificationsSelected(false);
+      })
+      .catch((e) => {
+        console.error('failed to update notification read status', e);
+      });
+  };
+
+  const selectAllNotifications = (selected: boolean) => {
+    updateAllNotificationsSelected(selected);
+  };
+
+  const selectVisibleNotifications = () => {
+    const visibleNotifications =
+      activeFilters.length > 0 ? filteredNotifications : notifications;
+    visibleNotifications.forEach((notification) =>
+      updateSelectedNotification(notification.id, true)
+    );
+  };
+
+  const onFilterSelect = (chosenFilter: string) => {
+    activeFilters.includes(chosenFilter)
+      ? setActiveFilters(
+          activeFilters.filter((filter) => filter !== chosenFilter)
+        )
+      : setActiveFilters([...activeFilters, chosenFilter]);
+  };
+
+  const onNavigateTo = (link: string) => {
+    navigate(link);
+    onNotificationsDrawerClose();
+  };
+
+  const dropdownItems = [
+    <DropdownItem key="actions" description="Actions" />,
+    <DropdownItem
+      key="read selected"
+      onClick={() => {
+        onUpdateSelectedStatus(true);
+      }}
+      isDisabled={notifications.length === 0}
+    >
+      Mark selected as read
+    </DropdownItem>,
+    <DropdownItem
+      key="unread selected"
+      onClick={() => {
+        onUpdateSelectedStatus(false);
+      }}
+      isDisabled={notifications.length === 0}
+    >
+      Mark selected as unread
+    </DropdownItem>,
+    <Divider key="divider" />,
+    <DropdownItem key="quick links" description="Quick links" />,
+    <DropdownItem
+      key="notifications log"
+      onClick={() => onNavigateTo('/settings/notifications/notificationslog')}
```

Comment:
Same as with the `to` links. I think we can remove the `/settings` URL. But I'd have to check this locally to see what can remain and what can be removed.